aif360/sklearn/datasets/openml_datasets.py

```diff
@@ -219,9 +221,8 @@ def fetch_bank(*, data_home=None, cache=True, binary_age=True, percent10=False,
         >>> bank_num.X.shape
         (45211, 6)
     """
-    # TODO: this seems to be an old version
-    df = fetch_openml(data_id=1558 if percent10 else 1461, data_home=data_home
-                      or DATA_HOME_DEFAULT, cache=cache, as_frame=True).frame
+    store = open_ml_store()
+    df = store.download(None, None)
```

Comment:
Instead of none, lets set these to the inputs, so `(data_id=1558 if percent10 else 1461, data_home=data_home)`

Reply:
Ok